frontends/aiger/aigerparse.cc

```diff
@@ -1062,7 +1062,7 @@ struct AigerFrontend : public Frontend {
 		if (module_name.empty()) {
 #ifdef _WIN32
 			char fname[_MAX_FNAME];
-			_splitpath(filename.c_str(), NULL /* drive */, NULL /* dir */, fname, NULL /* ext */)
+			_splitpath(filename.c_str(), NULL /* drive */, NULL /* dir */, fname, NULL /* ext */);
 				module_name = fname;
```

Comment:
Oops. Looks like I did an extra tab here. Would you be able fix that too whilst you're in the region? Thanks!